src/sagemaker/fw_utils.py

```diff
@@ -638,6 +639,36 @@ def warn_if_parameter_server_with_multi_gpu(training_instance_type, distribution
         logger.warning(PARAMETER_SERVER_MULTI_GPU_WARNING)
 
 
+def profiler_config_deprecation_warning(
+    profiler_config, image_uri, framework_name, framework_version
+):
+    """Put out a deprecation message for if framework profiling is specified TF >= 2.12 and PT >= 2.0"""
+    if profiler_config is None or profiler_config.framework_profile_params is None:
+        return
+
+    if framework_name not in ("pytorch", "tensorflow"):
```

Comment:
Can we put this info in some easy to update config? including the version info?

Reply:
Not sure what you mean and if it is worth it. It is a one time shot for us, we are getting out of framework profiling.